sdl/v2.go

```diff
@@ -178,6 +179,11 @@ func (sdl *v2) Manifest() (manifest.Manifest, error) {
 
 			for _, expose := range svc.Expose {
 				for _, to := range expose.To {
+					if expose.Proto == "" {
+						expose.Proto = manifest.TCP
+					} else {
+						expose.Proto = strings.ToUpper(expose.Proto)
```

Comment:
I think we should have a `manifest.ParseProto()` that returns an enum or type derived from `string`.